scripts/applyKube.sh

```diff
@@ -0,0 +1,11 @@
+retryCount=0
+workDir=$1
+kubeConfig=$2
+pushd $workDir
+until [ "$retryCount" -ge 5 ]
+do
+   kubectl apply -k . $kubeConfig  && break  
```

Comment:
Interesting way to break out of the loop, my shell programming is way far behind but that looks quite idiomatic to the language.